build/webpack/webpack.datascience-ui.config.builder.js

```diff
@@ -121,12 +121,60 @@ function buildConfiguration(bundle) {
     // console.error(`Bundle = ${ bundle }`);
     // Folder inside `webviews/webview-side` that will be created and where the files will be dumped.
     const bundleFolder = bundle;
+    const filesToCopy = [];
+    if (bundle === 'ipywidgetsRenderer' || bundle === 'ipywidgetsKernel') {
+        filesToCopy.push({
+            from: path.join(
+                constants.ExtensionRootDir,
+                'node_modules',
+                '@vscode',
+                'jupyter-ipywidgets7',
+                'dist',
+                'ipywidgets.js'
+            ),
+            to: path.join(
+                constants.ExtensionRootDir,
+                'out',
+                'node_modules',
+                '@vscode',
+                'jupyter-ipywidgets7',
+                'dist',
+                'ipywidgets.js'
+            )
+        });
+        filesToCopy.push({
```

Comment:
I assume this is supposed to be in an `else` block?

Reply:
no, its still here, 
just made it separate to highlight the two sets, one for 7 and one for 8